cpplint/cpplint.py

```diff
@@ -5717,6 +5717,11 @@ def CheckGflagDefaultApi(filename, clean_lines, linenum, error):
     linenum: The number of the line to check.
     error: The function to call with any errors found.
   """
+
+  # flag_tags.h is the only file allowed to use default gflag macros.
+  if filename.endswith("src/yb/util/flags/flag_tags.h"):
```

Comment:
For backport branches, both the adding macros e649ffd81edac569dbad713826593cd2d1d11c8b and move to flags subdir 95e6155d732fe64a02ae04969c4616a951d89a43 will be needed for this to work.  If you change it to "flag_tags.h" like in #9, it will work even if commit 95e6155d732fe64a02ae04969c4616a951d89a43 is omited in the backport branch.

Reply:
These rules are disabled in older branch so we dont have to worry about them
https://phabricator.dev.yugabyte.com/D20793

I will later update it to the new flag_validator name